app/src/main/java/edu/uc/kovaciad/slicetracker/RetrofitInstance.kt

```diff
@@ -5,7 +5,7 @@ import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitInstance {
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "127.0.0" // TODO: Replace with our webapp (Sprint 2)
+    private const val BASE_URL = "127.0.0" // TODO: Replace with our webapp (Sprint 2)
```

Comment:
Added const as the base url will not change. https://kotlinlang.org/docs/coding-conventions.html#property-names